perllib/FixMyStreet/DB/ResultSet/Problem.pm

```diff
@@ -171,18 +174,18 @@ sub _recent {
         $attrs->{bind} = [ $lat, $lon, $dist ];
         $attrs->{join} = 'nearby';
         $probs = [ mySociety::Locale::in_gb_locale {
-            $rs->search( $query, $attrs )->all;
+            sort { $b->confirmed <=> $a->confirmed } $rs->search( $query, $attrs )->all;
```

Comment:
I think this will show warnings on Zurich where public reports don't need a confirmed to be set, so might need to default to `($b->confirmed || 0) <=> ...`. Or not bother with sorting in code at all :)